packages/next/src/server/app-render/app-render.tsx

```diff
@@ -418,6 +439,27 @@ async function ReactServerApp({
     missingSlots,
   })
 
+  const initialFlightData = await walkTreeWithFlightRouterState({
```

Comment:
I'm not crazy about copy-pastaing this giant function call with a slight modification, but also didn't want to go overboard with refactoring these utils. 🤔